config/database.yml

```diff
@@ -20,8 +20,8 @@ default: &default
   # For details on connection pooling, see rails configuration guide
   # http://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  user: postgres
-  host: <%= ENV['POSTGRES_HOST'].present? ? ENV['POSTGRES_HOST'] : 'operationcode-psql' %>
+  user: <%= ENV['POSTGRES_USER'] %>
```

Comment:
It seems like you'd want something like `ENV['POSTGRES_USER'].presence || 'postgres'` to keep the current behavior.

Reply:
The current behavior is kept by the `.env` file which is applied when running `docker-compose`